package-overrides/github/sytelus/cryptojs@3.1.2.json

```diff
@@ -0,0 +1,129 @@
+{
+  "main": "core.js",
+  "format": "global",
+  "directories": {
+    "lib": "components"
+  },
+  "shim": {
+    "core.js": {
+      "exports": "CryptoJS"
+    },
+    "sha224.js": {
+      "deps": [
+        "cryptojs/sha256.js"
+      ]
```

Comment:
Certainly it would be enough to make it work but it might be quite nice to
get back the direct module value for sha224 instead of the same global
object for each independent import.
On Tue, 15 Mar 2016 at 08:41, Damien Lebrun <notifications@github.com>
wrote:

> In package-overrides/github/sytelus/cryptojs@3.1.2.json
> <https://github.com/jspm/registry/pull/815#discussion_r56100085>:
>
> > @@ -0,0 +1,129 @@
> > +{
> > +  "main": "core.js",
> > +  "format": "global",
> > +  "directories": {
> > +    "lib": "components"
> > +  },
> > +  "shim": {
> > +    "core.js": {
> > +      "exports": "CryptoJS"
> > +    },
> > +    "sha224.js": {
> > +      "deps": [
> > +        "cryptojs/sha256.js"
> > +      ]
>
> It uses CryptoJS global reference to add a SHA224 item to CryptoJS.algo.
> It doesn't export CryptoJS.algo.SHA224; it doesn't even export CryptoJS
> if it's not present.
>
> Requiring core.js (via "*": {"deps": ["cryptojs/core.js"]}) which exports
> CryptoJS is not enough for SystemJS?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/jspm/registry/pull/815/files#r56100085
>